docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,69 @@
-  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
-  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
+1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
+1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
 
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Editor’s Game view, select **Play**.
+1. In the Editor’s Game view, select **Play**.
 
     > You can change the resolution of the Game view in your Unity Editor to make sure it does not appear stretched on your mobile device. Choose the resolution that’s identical to your mobile device to produce the best results.
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+1. You should now see your Unity Editor game view mirrored on your Android device.
 
 ## Connecting your Android emulator to a local deployment
 
-  1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
+1. [Start your Android Emulator in Android Studio.](https://developer.android.com/studio/run/managing-avds)
 
     > Ensure to choose the same CPU architecture for your virtual machine as your development computer. If you don’t, you will get warning messages as mis-matched CPU architecture affects performance.
+1. [Setup your local deployment.](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start the local deployment from the Unity Editor though **SpatialOS** > **Local launch**.
```

Comment:
"through" instead of "though"

"your local deployment" instead of "the local deployment" -> just makes it a bit clearer that this is happening on their side

Reply:
<3 <3 <3

Reply:
Have added this and another suggestion below. I would probably use "your Unity Editor" when writing but I note that most of this doc says "the Unity Editor". Either is fine but "your Unity Editor" sounds more comfortable as it's your version on your machine.